lib/gems/pending/appliance_console/cli.rb

```diff
@@ -322,6 +329,25 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def set_server_state
+      service = LinuxAdmin::Service.new("evmserverd")
+      service_running = service.running?
+      case options[:server]
+      when :status
+        service_running ? "running" : "not running"
+        say "EvmServed is " + service.running?
+      when :start
+        service.start unless service_running
+      when :stop
+        service.stop if service_running
+      when :restart
+        say "EvmServed is restarting"
+        service.restart
+      else
+        say "wrong server state"
```

Comment:
I think `Invalid server action` would be better as the user is writing up actions rather than states.